src/legacy/core_plugins/kibana/public/discover/doc_table/_doc_table.scss

```diff
@@ -1,3 +1,4 @@
+@import './_doc_table_source_mixin.scss';
```

Comment:
Just to follow our established patterns for mixin files, can you:
1. move this new file to the [top level discover folder](https://github.com/elastic/kibana/tree/ab40af16a43759c2976c8372b7da6b717f7e9723/src/legacy/core_plugins/kibana/public/discover),
2. rename it to `_mixins.scss`
3. import the mixins file in [line 2 of the `_index.scss` file](https://github.com/elastic/kibana/blob/ab40af16a43759c2976c8372b7da6b717f7e9723/src/legacy/core_plugins/kibana/public/discover/_index.scss). Also when importing sass files you can drop the leading underscore and the file extension so just: `@import './mixins';`

Reply:
done, thank you for introduction into this pattern 👍